client/Lykke.Service.FeeCalculator.Client/FeeCalculatorClient.cs

```diff
@@ -115,5 +114,110 @@ public void Dispose()
 
             throw new Exception(ApiError);
         }
+
+        public async Task AddFeeAsync(FeeModel fee)
+        {
+            var response = await _service.AddFeeAsync(fee);
+
+            if (response is ErrorResponse error)
+            {
+                await _log.WriteErrorAsync(nameof(FeeCalculatorClient), nameof(AddFeeAsync),
```

Comment:
Not sure that the client should it self write a error message, the caller must take care about this